src/components/Atoms/Alert/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import styled from "styled-components";
+
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import BREAKPOINT from "../../../variables/breakpoint";
+import FONT_FAMILY from "../../../variables/font_family";
+
+export const Alert = ({ isActive, errorText = "" }) => {
+  return <StyledText isActive={isActive}>{errorText}</StyledText>;
+};
+
+const StyledText = styled.div`
+  position: absolute;
+  top: 80px;
+  left: 50%;
+  transform: translateX(-50%);
+  background-color: ${COLOR.RED};
+  max-width: 360px;
+  width: 100%;
+  height: 20px;
+  border-radius: 4px;
+  padding: 10px 20px;
+  color: ${COLOR.WHITE};
+  ${TEXT.S};
+  font-family: ${FONT_FAMILY.NOTO_SANS};
+  opacity: ${(isActive) => (isActive ? 1 : 0)};
```

Comment:
```suggestion
 opacity:0;
  ${(props) =>
    props.isActive &&
    `{opacity:1 ; transform: translateX(-50%) translateY(0)}`};
```
このように書くことで、フェードイン・アウトが出来ます。
&& は論理積といいます。
https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Operators/Logical_AND

![スクリーンショット (55)](https://github.com/Yuseidosukoiya/Yusei-frontend-basic-2022-todoapp-2/assets/137466167/f937d31e-ba7e-4394-9cc0-6a9c299857b7)
教科書の例ではCSSプロパティは｛｝の外にありますが、中に書くこともできます。